examples/juniper/cptx-ixia/cptx-ixia.pb.txt

```diff
@@ -0,0 +1,110 @@
+name: "ixia-c"
+nodes: {
+    name: "otg"
+    vendor: KEYSIGHT
+    version: "0.0.1-3423-local"
+    services: {
+        key: 443
+        value: {
+            name: "ixia-c"
+            inside: 443
+            outside: 0
+        }
+    }
+    services: {
+        key: 40051
+        value: {
+            name: "grpc"
+            inside: 40051
+            outside: 0
+        }
+    }
+    services: {
+        key: 50051
+        value: {
+            name: "gnmi"
+            inside: 50051
+            outside: 0
+        }
+    }
+}
+
+nodes: {
+    name: "cptx1"
```

Comment:
nit: this can just be cptx given thats its the only cptx node

Reply:
Done